schoolopy/main.py

```diff
@@ -1891,15 +1893,44 @@ def get_assignment_comment(self, section_id, assignment_id, comment_id):
     # TODO: Support Web Content Package
     # TODO: Support Completion
 
+    def get_section_grades(self, section_id, query_name=None, query_value=None):
+        """
+        Returns a list of grades (paged).
+
+        :param section_id: ID of course's section.
+        :return: a list of grades for that section.
+
+        The following optional query strings can be appended to the path
+        to filter results:
+
+        :param query_name (timestamp, assignment_id, or enrollment_id)
+        :param query_value
+
+        timestamp:      return only grades that have been changed since
```

Comment:
I'm not sure "query" is the best name or the best way to go about this. Why not specifically ask for a timestamp?